x/mongo/driver/driver.go

```diff
@@ -54,6 +54,9 @@ type Server interface {
 
 	// RTTMonitor returns the round-trip time monitor associated with this server.
 	RTTMonitor() RTTMonitor
+
+	// IsCryptd represents whether the server is a mongocryptd process.
+	IsCryptd() bool
```

Comment:
Suggest removing `IsCryptd()` from the `driver.Server` interface. The `isCryptd` field can be checked from the `description.Server` on the connection.

Reply:
Ah great point! Extending the `Server` interface is a pain.